packages/frontend/src/features/reporting/components/AdjudicateNotification/AdjudicateNotification.tsx

```diff
@@ -75,25 +77,63 @@ function useActiveAdjudication() {
 
 export default function AdjudicationNotification() {
     const { data: activeAdjudication, refetch } = useActiveAdjudication()
-
     const [open, toggle] = useToggle(false)
+    const [confirmOpen, setConfirmOpen] = useState(false)
+    const [buttonVisible, setButtonVisible] = useState(true)
 
-    const onClose = () => {
-        refetch()
+    useEffect(() => {
+        if (activeAdjudication) {
+            setButtonVisible(true)
+        }
+    }, [activeAdjudication])
+
+    const closeAdjudicate = () => {
         toggle(false)
     }
```

Comment:
The button should be visible again when closing adjudication dialog.

Reply:
The button's visibility depends on `buttonVisible`, so it should be visible again when closing adjudication dialog now. Please let me know if i misunderstood.

Reply:
The floating button becomes invisible if I close adjudication dialog. You should make it visible again.

https://github.com/user-attachments/assets/d5d3c21f-288c-43e9-b56e-a5bc7b433dc9

Reply:
Got it ! Solved.